### ISSUE TITLE
DexYCB dataset and pretrained models

### ISSUE THREAD

**Issue by hnuzhy** [patienceFromZhou/simpleHand#15]
Thank you for releasing your excellent work. I would like to ask if there is a plan to open source the preprocessed DexYCB dataset and the model weights trained on it? If so, when will it be released? Because DexYCB is more complex than FreiHAND, the resulting light-weight model has better generalizability and efficiency in real application. The current SOTA methods [HaMeR](https://github.com/geopavlakos/hamer) and [WiLoR](https://github.com/rolpotamias/WiLoR) are both based on ViT-H, which is too large to be user-friendly. Your work will be a better choice for this community!

**Comment by LiQiu0224** [comment]
Hello, I encountered the same problem. May I ask if you have found a solution?

**Comment by yzyzyer** [comment]
I trained and tested on the dexycb dataset using S0 split(which has 406888 images for training and  98954 images for evaluation) and replaced the original normalization with ImageNet normalization
`image = image / 255.0 - 0.5` ->
`mean = torch.tensor([0.485, 0.456, 0.406], dtype=torch.float32,
                             device=image.device).view(1, 3, 1, 1)
std  = torch.tensor([0.229, 0.224, 0.225], dtype=torch.float32,
                             device=image.device).view(1, 3, 1, 1)
image = (image / 255.0 - mean) / std`
My results are listed as follows:
Evaluation 3D KP results:
auc=0.757, mean_kp3d_avg=1.24 cm
Evaluation 3D KP ALIGNED results:
auc=0.897, mean_kp3d_avg=0.51 cm

Evaluation 3D MESH results:
auc=0.742, mean_kp3d_avg=1.31 cm
Evaluation 3D MESH ALIGNED results:
auc=0.900, mean_kp3d_avg=0.50 cm

F-scores
F @ 5.0mm = 0.4504        F_aligned @ 5.0mm = 0.8156
F @ 15.0mm = 0.9171       F_aligned @ 15.0mm = 0.9924

**Comment by sanshengyan0** [comment]
> 我在dexycb数据集上使用S0分割进行训练和测试（训练时有406888张图片，评估时有98954张图片），并用ImageNet归一化替换了原始归一化 ->结果如下：评估3D KP结果：auc=0.757， mean_kp3d_avg=1.24厘米 评估3D KP对齐结果：auc=0.897， mean_kp3d_avg=0.51厘米`image = image / 255.0 - 0.5``mean = torch.tensor([0.485, 0.456, 0.406], dtype=torch.float32, device=image.device).view(1, 3, 1, 1) std = torch.tensor([0.229, 0.224, 0.225], dtype=torch.float32, device=image.device).view(1, 3, 1, 1) image = (image / 255.0 - mean) / std`
> 
> 评估三维网格结果：auc=0.742，mean_kp3d_avg=1.31厘米 评估三维网格对齐结果：auc=0.900，mean_kp3d_avg=0.50厘米
> 
> F分数 F @ 5.0mm = 0.4504 F_aligned @ 5.0mm = 0.8156 F @ 15.0mm = 0.9171 F_aligned @ 15.0mm = 0.9924

您好，您的工作非常优秀，能麻烦提供一下训练dexycb的文件吗

**Comment by yzyzyer** [comment]
> > 我在dexycb数据集上使用S0分割进行训练和测试（训练时有406888张图片，评估时有98954张图片），并用ImageNet归一化替换了原始归一化 ->结果如下：评估3D KP结果：auc=0.757， mean_kp3d_avg=1.24厘米 评估3D KP对齐结果：auc=0.897， mean_kp3d_avg=0.51厘米`` image = image / 255.0 - 0.5``mean = torch.tensor([0.485, 0.456, 0.406], dtype=torch.float32, device=image.device).view(1, 3, 1, 1) std = torch.tensor([0.229, 0.224, 0.225], dtype=torch.float32, device=image.device).view(1, 3, 1, 1) image = (image / 255.0 - mean) / std ``
> > 评估三维网格结果：auc=0.742，mean_kp3d_avg=1.31厘米 评估三维网格对齐结果：auc=0.900，mean_kp3d_avg=0.50厘米
> > F分数 F @ 5.0mm = 0.4504 F_aligned @ 5.0mm = 0.8156 F @ 15.0mm = 0.9171 F_aligned @ 15.0mm = 0.9924
> 
> 您好，您的工作非常优秀，能麻烦提供一下训练dexycb的文件吗

您好，可以参考我创建的fork: https://github.com/yzyzyer/simpleHand/tree/dex-ycb
主要方法
1.参考dexycb toolkit正确预处理dexycb数据集，训练时使用S0划分
2.在dataset.py的HandDataset类中首先根据真实的bounding box把原图crop到224*224的尺寸，这点很重要因为dexycb图片的原始尺寸比较大，手在整张图片中占的比例很小，直接把整张图片输入网络中很难学习到手部的信息
3.在hand_net.py中把普通归一化替换为ImageNet归一化
